### ISSUE TITLE
When printing a blog post, the menu overlays the text

### ISSUE THREAD

**Issue by jdamerow** [dh-tech/dh-tech.github.io#96]
E.g. when I try to print our whitepaper, some text is missing because it's hidden by the menu.

**Comment by rlskoeser** [comment]
Yeah, I see. We need to add some print css to hide the navbar and the floating jump to top link.  

Seems fine to just hide them entirely, right?

**Comment by rlskoeser** [comment]
resolved in #105
